docs/modules/vector-search/pages/vector-collections.adoc

```diff
@@ -0,0 +1,498 @@
+= Vector Collection
+
+== Intro
+
+The primary object for interacting with the database is a Vector Collection. This collection holds information about the vectors and metadata (value).
```

Comment:
I don't think we use "database" in context of vector collection, this is misleading.

The intro could contain more info about vector collection is used for (similarity search), what is the structure (key, value, vectors+indexes) - that is partially described.

Reply:
97e456281191372558f94773bdb743a9c1f92e4f